src/core/i18n/en/translation.en.json

```diff
@@ -1933,6 +1933,7 @@
         },
         "notifications": {
           "space-created": "Space created successfuly.",
+          "errorCreatingSpace": "Error creating Space.",
```

Comment:
It could go like that but the wording could be improved:
"An error occurred while creating the Space."